src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsLanguageDebugInfo.cs

```diff
@@ -96,13 +102,16 @@ public int GetNameOfLocation(IVsTextBuffer pBuffer, int iLine, int iCol, out str
                                             // NOTE(cyrusn): We have to wait here because the debuggers' 
                                             // GetNameOfLocation is a blocking call.  In the future, it 
                                             // would be nice if they could make it async.
-                                            var debugLocationInfo = _languageDebugInfo.GetLocationInfoAsync(document, point, cancellationToken).WaitAndGetResult(cancellationToken);
-
-                                            if (!debugLocationInfo.IsDefault)
+                                            _threadingContext.JoinableTaskFactory.Run(async () =>
                                             {
-                                                name = debugLocationInfo.Name;
-                                                lineOffset = debugLocationInfo.LineOffset;
-                                            }
+                                                var debugLocationInfo = await _languageDebugInfo.GetLocationInfoAsync(document, point, cancellationToken).ConfigureAwait(false);
```

Comment:
Was this needed as a part of this PR, or just general cleanup? (It's not bad, just not sure if there's some deep reason for it appearing here and I'm missing it.)

Reply:
i was changing TryMapSpanFromSecondaryBufferToPrimaryBuffer to be async, and that was called from this code.  so i JTF cleaned this all up.